com.unity.render-pipelines.universal/Runtime/Debug/DebugDisplaySettingsRendering.cs

```diff
@@ -42,6 +42,23 @@ public DebugWireframeMode wireframeMode
             }
         }
 
+        bool m_Overdraw = false;
+
+        /// <summary>
+        /// Whether debug overdraw mode is active.
+        /// </summary>
+        [Obsolete("overdraw has been deprecated. Use overdrawMode instead.", false)]
```

Comment:
When was this obsolete?

Reply:
I added it back. It was removed here #5883 
If I can see this correctly.